File: src/datahub/examples/python/sample_load/sample_load.py
#!/usr/bin/python

from client.python.dh_client import DataHubClient
from datahub.constants import *

'''
@author: anant bhardwaj
@date: Oct 11, 2013

Sample python code accesing DataHub APIs
'''


# conect to DataHub
client = DataHubClient('localhost', 9000)
con_params = DHConnectionParams(user='postgres', password='postgres', database=DHDatabase(name='postgres'))
con = client.connect(con_params=con_params)

result = client.load(con=con, url='demo.person')

for row in result.data.table.rows:
  print '\t'.join([c.value for c in row.cells])
  print '\n'

File: src/datahub/examples/javascript/sample_load.js


'''
@author: anant bhardwaj
@date: Oct 11, 2013

Sample javascript code accesing DataHub APIs
'''


var transport = new Thrift.Transport("/thrift/service/tutorial/");
var protocol  = new Thrift.Protocol(transport);
var client    = new CalculatorClient(protocol);

var version = client.get_version()
console.log(version);

